sdk/communication/azure-communication-administration/swagger/README.md

```diff
@@ -44,6 +44,35 @@ These settings apply only when `--tag=phonenumber` is specified on the command l
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/257f060be8b60d8468584682aa2d71b1faa5f82c/specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/preview/2020-07-20-preview1/communicationservicesadministration.json
 override-client-name: PhoneNumberAdminClient
 ```
+### Rename searchId to reservationId
+
+
+
+``` yaml
+
+directive:
+
```

Comment:
seems that there are a few extra new lines around

Reply:
fixed